docs/50-sdk.md

```diff
@@ -388,22 +388,44 @@ Find some samples and getting stated tutorial in the `examples/sdk/` directory o
     The Batch, Storage, Docker and SSH secrets used to create AZTK clusters. For more help with setting these values see [Getting Started](./00-getting-started.md).
 
     Fields:
+        service_principal: ServicePrincipalConfiguration
+        shared_key: SharedKeyConfiguration
+        docker: DockerConfiguration
 
+        ssh_pub_key: str
+        ssh_priv_key: str
```

Comment:
Might be worth mentioning that exactly one of `service_principal` and `shared_key` must be provided.